src/kixi/nhs/board_report.clj

```diff
@@ -116,7 +222,7 @@
          data        (storage/get-resource-data ckan-client resource_id)]
     (patient-experience-of-gp-services recipe data)))
 
-(defn process-patient-experience-recipes [ckan-client recipes]
+(defn patient-experience-ethnicity-analysis [ckan-client recipes]
```

Comment:
Not sure what you mean.. i've renamed that function to `patient-experience-ethnicity-analysis` so that it matches `patient-experience-deprivation-analysis`. These recipes both work on patient experience data, but they analyse different factors.